src/python/ensembl/io/genomio/manifest/compute_stats.py

```diff
@@ -69,7 +69,8 @@ class manifest_stats:
     def __init__(self, manifest_dir: str, accession: Optional[str], datasets_bin: Optional[str]):
         self.manifest = f"{manifest_dir}/manifest.json"
         self.accession: Optional[str] = accession
-        self.error = False
+        self.errors: List[str] = []
+        self.errors_file = f"{manifest_dir}/stats_diff.json"
```

Comment:
```suggestion
        self.errors_file = Path(manifest_dir) / "stats_diff.log"
```
I may be wrong, but it looks like you are printing simple text inside this file.

Reply:
Indeed, good catch